bin/load_test

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+
+fancy_echo() {
+  local fmt="$1"; shift
+
+  # shellcheck disable=SC2059
+  printf "\n$fmt\n" "$@"
+}
+
+brew_install_or_upgrade() {
+  if brew_is_installed "$1"; then
+    if brew_is_upgradable "$1"; then
+      fancy_echo "Upgrading %s ..." "$1"
+      brew upgrade "$@"
+    else
+      fancy_echo "Already using the latest version of %s. Skipping ..." "$1"
```

Comment:
I have not seen this `fancy_echo`/`printf` pattern before in bash, why not just interpolate?

```
"Already using the latest version of $1. Skipping ..."
```